.env.local

```diff
@@ -19,8 +19,9 @@ ES_BACKUPS_PATH=/tmp/backups
 # Omang Service
 ###############
 # Example : OMANG_DEV_MOUNT_FOLDER=/Users/lapin/Workspace/Jembi/hie-botswana/projects/omang-service-mediator
-OMANG_DEV_MOUNT_FOLDER=/home/hexastack/Desktop/hie-botswana/projects/omang-service-mediator
+OMANG_DEV_MOUNT_FOLDER=
 # For MacOs : build locally and use OMANG_ORACLE_DOCKER_IMAGE=oracle/database:19.3.0-ee-slim-faststart
-OMANG_ORACLE_DOCKER_IMAGE=gvenzl/oracle-free  
+# FOr Linux : build locally and use OMANG_ORACLE_DOCKER_IMAGE=gvenzl/oracle-free   
+OMANG_ORACLE_DOCKER_IMAGE=
```

Comment:
`OMANG_ORACLE_DOCKER_IMAGE=gvenzl/oracle-free` by default, it should be the linux one